compiler-plugin/src/main/java/io/ballerina/stdlib/graphql/compiler/diagnostics/DiagnosticMessage.java

```diff
@@ -26,50 +26,51 @@
+    ERROR_119("the graphql:Upload cannot be used as an input type of resource method ''{0}''"),
+    ERROR_120("multidimensional graphql:Upload array parameter found in method ''{0}''. GraphQL input cannot have "
                       + "multidimensional graphql:Upload arrays"),
-    ERROR_121("Graphql input type must not be a subtype of `error?`"),
-    ERROR_122("Invalid union type for GraphQL input type"),
-    ERROR_123("Non-distinct service class `{0}` is used as a GraphQL interface implementation"),
-    ERROR_124("Invalid hierarchical resource path `{0}` found in subscribe resource"),
-    ERROR_125("Invalid return type `{0}` found in subscribe resource `{1}`. GraphQL subscribe resource must return "
-                      + "`stream` type"),
-    ERROR_126("Invalid GraphQL resource accessor `{0}` found in resource `{1}`. Only " + RESOURCE_FUNCTION_GET +
-                      " and " + RESOURCE_FUNCTION_SUBSCRIBE + " are allowed"),
-    ERROR_127("Failed to generate the schema from the service. {0}"),
-    ERROR_128("Invalid resource method `{0}` found in GraphQL interceptor. GraphQL interceptors can not have resource "
-                      + "methods"),
-    ERROR_129("Invalid remote method `{0}` found in interceptor service. Only \"execute\" remote method is allowed"),
-    ERROR_130("Anonymous record `{0}` cannot be used as the type of the field `{1}`"),
-    ERROR_131("Anonymous record `{0}` cannot be used as an input object type of the field `{1}`"),
-    ERROR_132("Invalid return type `{0}` provided for the GraphQL field `{1}`. `{0}` is not a service class");
+    ERROR_121("the Graphql input type must not be a subtype of '''error?'''"),
+    ERROR_122("invalid union type for GraphQL input type"),
+    ERROR_123("non-distinct service class ''{0}'' is used as a GraphQL interface implementation"),
+    ERROR_124("invalid hierarchical resource path ''{0}'' found in subscribe resource"),
+    ERROR_125("invalid return type ''{0}'' found in subscribe resource ''{1}''. GraphQL subscribe resource must return "
+                      + "'''stream''' type"),
+    ERROR_126("invalid GraphQL resource accessor ''{0}'' found in resource ''{1}''. Only '''" + RESOURCE_FUNCTION_GET
+                      + "''' and '''" + RESOURCE_FUNCTION_SUBSCRIBE + "''' are allowed"),
+    ERROR_127("failed to generate the schema from the service. {0}"),
+    ERROR_128("invalid resource method ''{0}'' found in GraphQL interceptor. GraphQL interceptors can not have resource"
+                      + " methods"),
+    ERROR_129("invalid remote method ''{0}'' found in interceptor service. Only \"execute\" remote method is allowed"),
+    ERROR_130("anonymous record ''{0}'' cannot be used as the type of the field ''{1}''"),
+    ERROR_131("anonymous record ''{0}'' cannot be used as an input object type of the field ''{1}''"),
+    ERROR_132("invalid return type ''{0}'' provided for the GraphQL field ''{1}''. ''{0}'' is not a service class");
```

Comment:
What's the reason for using two or three single quotes in the error messages?

Reply:
The single quote is used to represent a section within the message pattern that will not be formatted. 2/3 single quotes are used to escape the character.